docker/Dockerfile

```diff
@@ -3,7 +3,6 @@ FROM centos:centos6.6
 MAINTAINER FIWARE Orion Context Broker Team. Telefónica I+D
 
 ENV ORION_USER orion
-ENV GIT_REV_ORION_PROXYCOAP 749e22376664dd3adae17492090e58882d3b28a7
```

Comment:
@LeandroGuillen please check that docker building still works after the changes I have introduced (and that all the CAOP stuff has been removed and no leftover remains).